python-flask/docker-compose.yml

```diff
@@ -1,20 +1,17 @@
 services:
-  web:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    container_name: flask_app
+  python-flask:
+    image: waect/python-flask
+    build: .
+    container_name: python-flask
+    ports:
+      - "5000:500"
     env_file:
       - ../.env.local
```

Comment:
if this is the experiment one we should change it to .env.production (with our cloud db address)

Reply:
Yes I will add the secrets

Reply:
Secrets added!